### DISCUSSION TITLE
How to automate building Github page from all of my Github repository's readme.md file?

### DISCUSSION THREAD

**Discussion by quazi-irfan** [community/community#21956] [Other Feature Feedback, Questions, & Ideas]
<p>I want to showcase a list of my Github projects on my Github page, and I want to automate the process of building the Github page. I want the process to fetch the content of readme.md from each of my repositories, and place it in a template in a static website.</p>
<p>I was wondering if there already an established process for it before I start working on my custom solution?</p>
<ul>
<li>
</li>
</ul>

**Comment by birdcar** [comment, marked as answer]
<blockquote>
<p> </p>
</blockquote>
<blockquote>
<p>I was wondering if there already an established process for it before I start working on my custom solution? </p>
</blockquote>
<p>There isn’t an established process for doing this, but you’re welcome to use the API to build this for yourself. </p>
<p><a href="https://developer.github.com/v3/" rel="nofollow noopener">https://developer.github.com/v3/</a></p>
<p>If you manage to build something that does this, be sure to share a link to your work here in the community! I know that I’d be super interested in seeing how you do someting like this. </p>
